river/compose/pipeline.py

```diff
@@ -590,10 +593,14 @@ def transform_one(self, x: dict, **params):
         that precede the final step are assumed to all be transformers.
 
         """
+        print(f"{self._STATELESS=}")
         x, last_step = self._transform_one(x)
+
         if isinstance(last_step, base.Transformer):
-            if not last_step._supervised:
-                last_step.learn_one(x)
+            if not self._STATELESS:
+                if not last_step._supervised:
```

Comment:
```suggestion
            if not self._STATELESS and not last_step._supervised:
```

no?

Reply:
Sure, just wanted to keep it in line with how things a written in other methods :)